betel/classifier_data_set_builder.py

```diff
@@ -0,0 +1,61 @@
+import shutil
+import pathlib
+import pandas as pd
+from betel import utils
+from betel import info_files_helpers
+from betel import data_set_builder
+
+
+class ClassifierDataSetBuilder(data_set_builder.DataSetBuilder):
+    """A class for splitting classifier data into train-validation-test sets."""
+
+    def __init__(self, input_dir: pathlib.Path, storage_dir: pathlib.Path, classes: [str] = None):
+        """Constructor.
+
+        :param input_dir: data to be split (output of the scraper)
```

Comment:
I think it is not super relevant for Github projects, but the Google's style for documenting is different: go/pystyle#function-docs

Reply:
Unfortunately, I do not have access to that. I used [this styleguide](https://google.github.io/styleguide/pyguide.html#typing-imports) and some suggestions from Seb&Momo.